microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -159,218 +235,325 @@ If you publish a person's Personal Information to an outside service or third-pa
 
-### 2.10 Excessive Profanity and Inappropriate Content
+Your extension must adhere to applicable laws and regulations, Microsoft's terms of service, and applicable privacy policies. Microsoft reserves the right to review extensions for compliance and to promptly remove any noncompliant extension.
 
-*   Your extension must not contain excessive or gratuitous profanity.
 
-*   Your extension must not contain or display content that a reasonable person considers obscene.
+<!-- ------------------------------ -->
+#### 2.9 Illegal activity
 
-### 2.11 Country/Region Specific Requirements
+Your extension and the associated data must not contain content or functionality that encourages, facilitates, or glamorizes illegal activity in the real world.
 
-Content that is offensive in any country/region to which your extension is targeted is not allowed.  Content may be considered offensive in certain countries/regions because of local laws or cultural norms.  Examples of potentially offensive content in certain countries/regions include the following:
+
+<!-- ------------------------------ -->
+#### 2.10 Excessive profanity and inappropriate content
+
+* Your extension must not contain excessive or gratuitous profanity.
+
+* Your extension must not contain or display content that a reasonable person considers obscene.
+
+
+<!-- ------------------------------ -->
+#### 2.11 Country- or region-specific requirements
+
+Content that is offensive in any country or region to which your extension is targeted is not allowed.  Content may be considered offensive in certain countries or regions because of local laws or cultural norms.  Examples of potentially offensive content in certain countries or regions include the following:
 
 **China**
 
-*   Prohibited sexual content.
-*   Disputed territory or region references.
-*   Providing or enabling access to content or services that are illegal under applicable local law.
+* Prohibited sexual content.
+* Disputed territory or region references.
+* Providing or enabling access to content or services that are illegal under applicable local law.
+
 
-### 2.12 Age Ratings
+<!-- ------------------------------ -->
+#### 2.12 Age ratings
 
-#### 2.12.1 Mature Content
+<!-- todo: add content between headings -->
+
+
+###### 2.12.1 Mature content
 
 When you submit your extension to [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd), you must indicate whether your extension displays content that should be marked "Mature".  When determining the rating for your extension, consider all the content in your app, including user generated content and ads, and to the content that your extension links.  If you indicate that your extension doesn't contain any "Mature" content, you are responsible for maintaining the accuracy of this rating.
```

Comment:
```suggestion
When you submit your extension to Partner Center, you must indicate whether your extension displays content that should be marked "Mature".  When determining the rating for your extension, consider all the content in your app, including user generated content and ads, and to the content that your extension links.  If you indicate that your extension doesn't contain any "Mature" content, you are responsible for maintaining the accuracy of this rating.
```
instance 4

"Partner Center" occurs 5 times in this article.
4 times are linked - that's excessive linking; linking every mention for no reason.  Recommend un-linking some of the 4 instances.

And they are not a short Partner Center URL, they are a link that took a long time to resolve for me, and then rejected my account:
https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd

Reply:
@Bhuvana-Priya can we shorten the URL

Reply:
My results from URL lengths:

https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd - after a long time, i end up at https://partner.microsoft.com/en-us/dashboard/registration/AccountInfo?accountProgram=MSEdgeAddons "This account is not registered with Microsoft Edge program on Partner Center"
https://partner.microsoft.com/dashboard/microsoftedge/public/login - after a long time, i end up at https://partner.microsoft.com/en-us/dashboard/registration/AccountInfo?accountProgram=MSEdgeAddons "This account is not registered with Microsoft Edge program on Partner Center"

https://partner.microsoft.com/dashboard/microsoftedge/public - after a long time, i end up at https://partner.microsoft.com/en-us/dashboard/microsoftedge/

https://partner.microsoft.com/dashboard/microsoftedge - after a long time, i end up at https://partner.microsoft.com/en-us/dashboard/home
https://partner.microsoft.com/dashboard - after a long time, i end up at https://partner.microsoft.com/en-us/dashboard/home

https://partner.microsoft.com - quickly, end up at a regular webpage: https://partner.microsoft.com/en-US/  "Limitless innovation with Microsoft AI - Welcome to the Microsoft AI Cloud Partner Program."

Reply:
@Bhuvana-Priya @AbhisekhMSFT - 
In the Intro paragraph, I changed the Partner Center url from:
https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd
to:
https://partner.microsoft.com/dashboard/home
because that tagged URL matches where i end up, and that page contains some content for me instead of being empty.  It seems to be the best general-purpose link into Partner Center.

I un-linked the other mentions of Partner Center in this article.